tests/unit/rdf/test_resource_serialisation.py

```diff
@@ -314,6 +315,130 @@ class A(NewResource):
         URIRef("http://some-entity-uri"),
     ) in graph
 
+def test_markdown_datatype():
+    """ """
+
+    class A(NewResource):
+        p: Annotated[
+            str,
+            Triple(
+                URIRef("http://original-uri"),
+                PropertyStatus.recommended,
+                map_str_to_markdown,
+            ),
+        ]
+    a = A("http://some-a-uri")
+    a.p = "Hello, World"
+
+    graph = a.to_graph(Graph())
+
+    assert len(graph) == 2
```

Comment:
Does `len` give the depth of the graph? If not, is it possible to assert the depth of the graph as well?

Reply:
Graphs don't really have a depth, just the number of triples contained within them.